docs/installing.rst

```diff
@@ -33,6 +33,20 @@ To ensure it is installed propertly:
   libprotoc 3.6.0
 
 
+pandoc
+~~~~~~
+
+This generator relies on `pandoc`_ to convert from Markdown (the *lingua
```

Comment:
Nit: Fun Latin phrases mean ESL readers must understand 3 languages (native, English, pretty but dead) to read the docs. Consider replacing.